### ISSUE TITLE
Securing the future of Neblio and enabling the next era of development

### ISSUE THREAD

**Issue by happymoko** [NeblioTeam/Neblio-Improvement-Proposals#20]
### The Background

Unlike most other projects, the Neblio development team did not allocate tokens for themselves in the ICO, but bought into it like every other investor at that time. The whole development of the platform is funded solely by the team’s personal money. 

  Everyone who has been with Neblio over the years knows the team has always thought deeply on how to spend money on the development, marketing or other activities to not only generate the best ROI, but keep running costs low.

  Even with these constraints the team was able to build a platform that rivals and even trumps competitors spending tens of millions, not to mention there is already a growing ecosystem of 3rd party developers using Neblio to build their and their client’s projects. 

We are entering the phase of real world adoption here but unfortunately, this all is in danger.

### The Problem

Even with the restraint shown by the team in spending money, they weren’t able to sustain themselves from the staking rewards of their wallet alone when prices started declining and had to take additional tokens out at an ever increasing rate to cover bills .

In May alone the team had to liquidate Neblio in four transactions totaling about 110.000 tokens (30.000 on the 1st/2nd and 80.000 on the 12th of may). 

Even if we assume in the very best case that the amount on the 1st and 2nd of May was for the previous month, this would mean the team at these prices currently needs at least an average of 55.000 tokens monthly to sustain development and cover costs, most likely more than that. 
With 140.000 Neblios left in the team wallet it also becomes painfully clear how long this can continue if nothing is done to address the situation. 

### The Solution

To secure ongoing development of the platform, I propose an even 50/50 split of the staking rewards between the people staking and the team. This would lower the staking rewards from 10% to 5% yearly with the other 5% going directly to the team.

To make it clear from the beginning, this would only be a **temporary emergency measure** to make sure the development team has enough funds to continue and last only until the team is able to build funds up again to previous levels.

When this goal is achieved, the staking rewards should be changed again by a NIP vote, but this time to a ratio of 9% stakers and 1% team, making such a drastic change like the one proposed here unnecessary in the future, even if we encounter similarly harsh market conditions.

Having a reliable stream of 1% staking rewards available especially when the price improves would enable the additional funding of special projects like e.g. the UI overhaul of the Neblio wallet, the creation of specific content for the target audience of business customers/developers as well as retail investors and the financing of a marketing campaign offering onboarding or consulting and services packages (https://nebl.io/services) for free or at a reduced price to all qualifying enterprise customers.

The beautiful thing about this is that the implementation of the voting mechanism by the team last year makes this all possible in the first place. We as a community are now able to decide together and shape the future of this project. 

### The Percentage

Without knowing exactly how much of the 19M Neblios are being staked, we can at least estimate. The easiest way is to assume that all of the Top 100 wallets stake their tokens and look at the volume they represent. (a quick check seems to verify this assumption with the exception of the 2nd largest wallet).

Adding up all Top 100 Wallets (except the 2nd largest) shows that they represent about 81% of the total supply.

19M total supply of tokens with 80% tokens being staked = 15.2M of all tokens used for staking
15.2M tokens staked at 10% yearly  = 1.520.000 tokens as a reward generated yearly
50% of the rewards for the team = 760.000 tokens yearly = 63.000 monthly

Circling back to the problem paragraph above and the assumptions made in it, the 63.000 tokens seem to cover the monthly costs at the current price levels and with these rising allow the team to build up their war chest again until the community decides to implement the permanent change to a different ratio.

### Final Words

The solution proposed here is not mine, it is grown out of a discussion on the Neblio telegram channel between multiple community members trying to address the current situation and find the best long-term solution for the benefit of all involved. 

The Neblio team is very much part of this community and an important one at that. They have achieved incredible things already with even more exciting developments on the horizon (e.g. NeblioX) and we should make sure they are able to make the platform the best it can be.

### Proposal Voting Details

Voting details to be added after the discussion

### Other Information

This proposal should not be viewed as set in stone, but as an open invitation for debate to not only hear everyone in the community out, but to find the best possible solution together.

**Comment by NeblioTeam** [comment]
Opening up for public discussion and feedback. The Neblio Team does not take a stance for or against proposals before they are voted on, we simply provide information and guidance from a technical perspective. 

Several points, risks, and other technical information around this proposal:
- The tokenomics of Neblio have never been changed in its history, this would be a first. 
- This change would require a hard fork and a mandatory upgrade for Neblio Core.
- Even if this NIP passes the governance vote with support of stakers, there is no guarantee everyone will upgrade. Stakers that vote 'No' could refuse to upgrade Neblio Core, causing a fork in the chain and creating 2 blockchains, Neblio & Neblio Classic, for example. 
- In the event of a contentious fork like mentioned above, exchanges may or may not support the new chain, the choice would be up to each exchange.
- This would affect every single Neblio staker on the network, cutting the staking rewards they earn in exchange for validating network transactions in half. 
- We estimate this change would require a few weeks of development work and testing, plus 1 month to schedule the hard fork and give users enough time to upgrade Neblio Core. 

Due to this being an unprecedented change that would affect every single user on the network, we suggest that there be a detailed and lengthy discussion around this NIP, from many participants. 

If you support this NIP, do not support it, have questions, or have suggestions to improve it make your voice heard below before the proposal is locked in and a governance vote is scheduled👇

**Comment by adamkhalil1** [comment]
Since there are some risks involved which I am not a fan ( exchanges support and less decentralised mechanism) i would love to have more information from the team before voting especially because there is still nip19 which I would prefer to to vote yes for. How urgent are funds needed? At this price how much longer can the team afford further developing? Are there alternatives regarding funding. 

For me this information would be very important before voting on such big changes. But if there is no alternative I would not even care to spend 100% of my rewards. priority nr1 is to keep up the development.

**Comment by happymoko** [comment]
Agree, this is what I wrote in the telegram channel:

Awesome to see a lot of people discussing the current situation here and in the trading channel. Read through the NIP19 yesterday and actually really like it and think long-term it would be a great feature to be implemented. But if fear for the situation we are in right now, it will be too little, too late.

The premise is that the team will create a staking pool to get a percentage of the staking rewards to fund the ongoing development. The TOP100 Wallets represent 80% of all tokens staked. Binance represents 43% and they will definitely not put their coins in that. 

That leaves 37%, but how many of these wallets will move their coins to the team wallet when most of them have staked for years for themselves on Raspis etc. and could continue keeping all the rewards for themselves? Even with the absolute best case scenario and every single wallet staking with the team (which I believe is highly unlikely), 37% would still be not enough.

Another big problem is that the team has stated that it is a big development effort taking months to develop and many more to test thoroughly so we are looking at the end of the year for the release. And if my calculations are in the ballpark, there will be no further development by then. The implementation of the staking reward change was stated to only take a few weeks, which seems much more feasible.

**But the whole thesis rests on the observation that over the years the team wallet seemed to be the sole source of funding for the team and that this wallet is running out of funds fast. 

I get why the team does not want to influence the debate, but I think we need and deserve to know if this is really true. We are discussing to give up a significant portion of our rewards and how can we do that if we aren’t sure?**

**Comment by adamkhalil1** [comment]
I totally agree with you and you did some good math probably we really need those major changes for the short term but I am not a big fan of taking the risks. But bigger risk would be to run out of funds so both optuions I dont like :/. But as long as the development can keep going i i can live with that

**Comment by happymoko** [comment]
While I don't think a contentious split of the project into Neblio Classic and Neblio is very likely given that a chain without ongoing development looses much of it's appeal to retail investors and customers / developers, I definitely hope such drastic measures as described above are not needed. Maybe another solution like NIP19, the 9% / 1% split or something different is enough not only for the long-term funding, but for the current situation. 

As mentioned, it should be seen as a starting point for a discussion to proactively take on the current situation as it presents itself before it is too late. But we are pretty much speculating at this point and need input from the team. Cannot find the right / right-sized solution if we don't know the parameters.

**Comment by adamkhalil1** [comment]
Totally agree with you!
I think it makes a lot of sense to  put both NIPs on the roadmap, and yes if needed nip19 should be the highest priority

**Comment by Shane-Neblio** [comment]
answer is No

Why do you think Neblio team needs money from selling NEBLs?

They've always been drumming up business in the real world to use Neblio tech. they get funding from that.  They have stated this in the past. They earn money from clients in fiat

read:
https://nebl.io/services/

**Comment by Shane-Neblio** [comment]
because Neblio team relies on clients they have stated in the past that they are self-sustaining long into the future because of the demand for their expertise. Blockchain developers are highly sought after, especially if they can lay all of the foundation out for their clients. This often results in them selling the idea of Neblio tech being used as the backbone, which encourages constant development to improve Neblio technology.

Thus they do not need income from stake rewards.

**Comment by adamkhalil1** [comment]
As much as i want the neblio team not needing mone i dont think that they are overruned by clients regarding blockchain consultancy. I think in long therm that might become a thing but now i think many companies are either not ready yet or neblio marketing in this area has not been the highest priority. i think if their services were used that much the chain would have seen more adoption. But in long therm i can see this succeed!

**Comment by Shane-Neblio** [comment]
when i say Neblio team sells their talent & Neblio tech to clients, it may or may not be actually built on Neblio, depending on the clients needs.

sometimes a business may want to keep this private so there is no announcement or they may want their own isolated blockchain that has no gas-fee token, where they provide their own nodes or Neblio team deploys 10 nodes around the world for them.  They have a medium post where they breakdown their business model.

basically, Neblio team is moonlighting on the Neblio project to make ends meet and in they have said that this enables them to be able to sustain development of Neblio so they are not even close to in danger. 

It is likely those Team tokens you saw leaving that address is the members of the team each wanting their share after all these years.. which would explain the gradual disbursements so they can each do whatever they want with them.  I think their team is made up of about 5 people or so.
